internal/schemaversion/schemaversion.go

```diff
@@ -68,7 +68,7 @@ func String(sv *hcsschema.Version) string {
 func DetermineSchemaVersion(requestedSV *hcsschema.Version) *hcsschema.Version {
 	sv := SchemaV10()
 	if osversion.Get().Build >= osversion.RS5 {
-		sv = SchemaV10() // TODO: When do we flip this to V2 for RS5? Answer - when functionally complete. Templating. CredSpecs. Networking. LCOW...
+		sv = SchemaV21() // TODO: When do we flip this to V2 for RS5? Answer - when functionally complete. Templating. CredSpecs. Networking. LCOW...
```

Comment:
Are we okay with this switch? Is there a way to pass this info from the Sandbox.json rather than hardcoding?

Reply:
@jhowardmsft - Are we ready for this?
@madhanrm  - No we would like the platform to decide the best way to run the OCI spec not pass that down. IMO we always want to used the latest version supported if possible.

Reply:
Let me run the docker CI suite with this flipped before we do it.

Reply:
Lets remove the comment.